src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/resendtogaps/ListAssistHearingMessageHelper.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.resendtogaps;
+
+import static uk.gov.hmcts.reform.sscs.ccd.domain.HearingRoute.LIST_ASSIST;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.HearingState.CANCEL_HEARING;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.model.hearings.HearingRequest;
+import uk.gov.hmcts.reform.sscs.service.servicebus.HearingMessagingServiceFactory;
+
+@Service
+@RequiredArgsConstructor
+public class ListAssistHearingMessageHelper {
+
+    private final HearingMessagingServiceFactory hearingMessagingServiceFactory;
+
+    public void sendListAssistCancelHearingMessage(final String ccdCaseId) {
```

Comment:
I think it'd be better to pass in the required arguments so this method can be something like "sendMessage" - making it more generic